src/view/reviewManager.ts

```diff
@@ -522,10 +520,22 @@ export class ReviewManager implements vscode.DecorationProvider {
 			return;
 		}
 
-		this._telemetry.on('pr.checkout');
-		Logger.appendLine(`Review> switch to Pull Request #${pr.prNumber} - done`, ReviewManager.ID);
-		this.switchingToReviewMode = false;
-		await this._repository.status();
+		try {
+			this.statusBarItem.text = `$(sync~spin) Fetching additional data: pr/${pr.prNumber}`;
```

Comment:
This one can be skipped if the pull request has all information needed?

Reply:
It didn't seem obvious and quick to see if it has all the information needed. Maybe we can do another easy-win PR to early-out in `fullfillPullRequestMissingInfo` if it's all there?